app/views/tube_rack_summaries/show.html.erb

```diff
@@ -0,0 +1,55 @@
+<%= page_title("tube rack summary", @tube_rack.human_barcode) %>
+
+<div class='columnize'>
+  <%= bs_custom_panel(:info, :table, {class: 'table table-summary'}, {title: 'Summary' }) do %>
+        <tr>
+          <th>Submissions</th>
```

Comment:
No submissions on tube racks directly, so this section can go

Reply:
Yes, removing it